test/integration/test/new_webpush_test.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const chakram = require('chakram'), expect = chakram.expect;
+
+var Prepper = require('../lib/testPrepper.js');
+
+Prepper.makeSuite('Test Push Service locally',function(){
+
+  before(Prepper.foxboxManager.foxboxLogin);
+
+  it('Check webpush service is registered',function(){  
+    var pushService = false;
+
+    return chakram.get(Prepper.foxboxManager.serviceListURL)
+    .then(function(listResponse) {
+      expect(listResponse).to.have.status(200);
+      for (var entry of listResponse.body) {
+        if (entry.adapter === 'webpush@link.mozilla.org') {
+          pushService = true;                
+        }
+      }
+      expect(pushService).equals(true);
+    });
+  });
+
+  it('Add push subscription',function(){     
+    var pushURI = Prepper.webPush_server.getEndpointURI();
+    var pushkey = Prepper.webPush_server.getPublicKey();
+    var userAuth = Prepper.webPush_server.getUserAuth();
```

Comment:
It seems to be the only line added (compared to `test/integration/test/webpush_test.js`). I think we can have one single test files that deals with with both old and new webpush. For example:

```js
Prepper.makeSuite('Test Push Service locally',function(){

  var baseSubscriptionPayload = {
     'select': {
       'id':setter
     }, 
     'value': {
       'Json': {
         'subscriptions':[{
           'push_uri': Prepper.webPush_server.getEndpointURI(),
           'public_key': Prepper.webPush_server.getPublicKey()
         }]
      }
    }

  var newWebPushSubscriptionPayload = Object.assign({}, baseSubscriptionPayload, {
    value: {
       Json: {
          subscriptions: {
            auth: Prepper.webPush_server.getUserAuth();
          }
       }
    }
  });

   var testParams = [{
      suiteName: 'Old WebPush',
      payload: baseSubscriptionPayload
    }, {
      suiteName: 'New WebPush',
      payload: newWebPushSubscriptionPayload
    }];

   testParams.forEach(testParam => {
      describe(testParam.suiteName, function() {
         // do all the tests here.
      });
   })
  };
});
```